db/__tests__/seed.test.js

```diff
@@ -184,6 +184,59 @@ test('GET 200: responds with array of article objects in descending order', () =
         const { articles } = body
         expect(articles).toBeSortedBy('created_at',{ descending: true })
     })
+}) // topic, which filters the articles by the topic value specified in the query. If the query is omitted, the endpoint should respond with all articles.
+test('GET 200: responds with array of article objects matching the topic query', () => {
+    return request(app)
+    .get('/api/articles?topic=cats')
+    .expect(200)
+    .then(({ body }) => {
+        const { articles } = body
```

Comment:
We need a length check here again before the forEach to make sure we aren't getting an empty response back on this query